crates/bitwarden-crypto/src/enc_string/symmetric.rs

```diff
@@ -51,14 +53,21 @@ export type EncString = string;
 #[allow(unused, non_camel_case_types)]
 pub enum EncString {
     /// 0
-    AesCbc256_B64 { iv: [u8; 16], data: Vec<u8> },
+    AesCbc256_B64 {
+        iv: [u8; 16],
+        data: Vec<u8>,
+    },
     /// 1 was the now removed `AesCbc128_HmacSha256_B64`.
     /// 2
     AesCbc256_HmacSha256_B64 {
         iv: [u8; 16],
         mac: [u8; 32],
         data: Vec<u8>,
     },
+    // 7 The actual enc type is contained in the cose struct
+    XChaCha20_Poly1305_Cose_B64 {
+        data: Vec<u8>,
+    },
```

Comment:
Will 7 always be XChaCha or does it mean any COSE type?

Reply:
I'm fine keeping it as `XChaCha20_Poly1305_Cose_B64`, but the comment. feels misleading `The actual enc type is contained in the cose struct`.

Reply:
Renaming to `CoseEncrypt0`. The alg info is contained in the cose struct, and authenticated there. Note: Need to update the server side change to reflect this.

Also cc @MGibson1 since you had a different thread on this topic.